### PR TITLE
(wip) Update sponsorship packages to reflect sponsorship prospectus

### PR DESCRIPTION
**Pull request by despo**

### CODE REVIEW

File: _posts/2014-12-19-becoming-a-sponsor.md [hunk 1]
```diff
@@ -49,33 +49,52 @@ We also welcome monetary sponsorship to help us cover other expenses. The best w
...
+- Your logo listed on our Sponsors page under Silver tier
+- A shoutout to our 5400+ Twitter followers
+- A blog post dedicated to your company and their support of codebar
 
-*£100*
 
-- Your logo on our website
+*£1000*
+
+- 1 spots at a codebar workshop or event for your employees
```
Comment (KimberleyCook) [inline _posts/2014-12-19-becoming-a-sponsor.md]: spot instead of spots

File: _posts/2014-12-19-becoming-a-sponsor.md [hunk 2]
```diff
@@ -49,33 +49,52 @@ We also welcome monetary sponsorship to help us cover other expenses. The best w
...
-- Breakfast
-- Lunch
-- Continuous supply of coffee
-- Printed materials to help get the word out
-- Travel support to help students who live outside London get to the workshop
-- Stickers, t-shirts and other swag
-- After party refreshments
+- Promotional materials to help get the word out
+- Support for smaller codebar chapters that are at times unable to find local sponsorship
+- Paying for our infrastructure costs (we pay for most services we use, from hosting our event management application, to email)
```
Comment (KimberleyCook) [inline _posts/2014-12-19-becoming-a-sponsor.md]: I think we should put paying for our infrastructure first in this list. 

And we pay for for all services not most 😂

**Review by KimberleyCook — CHANGES_REQUESTED** [review]
Few little comments but other this is awesome, thanks for updating it.

File: _posts/2014-12-19-becoming-a-sponsor.md (hunk 1 above)
Comment (despo) [reply _posts/2014-12-19-becoming-a-sponsor.md]: Good spot, I did copy paste this from our existing sponsor prospectous as well 🤣

File: _posts/2014-12-19-becoming-a-sponsor.md (hunk 2 above)
Comment (despo) [reply _posts/2014-12-19-becoming-a-sponsor.md]: Good point, I was thinking that too. 👍 
One day, that will change 🤞

File: _posts/2014-12-19-becoming-a-sponsor.md (hunk 1 above)
Comment (KimberleyCook) [reply _posts/2014-12-19-becoming-a-sponsor.md]: oh dear 😂

**Review by KimberleyCook — APPROVED** [review]

**Pull request merged**